src/Session/Store.js

```diff
@@ -90,7 +90,7 @@ Store.unGuardPair = function (pair) {
   }
 
   /** if parsed do not re parse */
-  if (typeof (pair.d) === pair.t.toLowerCase()) {
+  if (typeof (pair.d) === pair.t.toLowerCase()) { // eslint-disable-line valid-typeof
```

Comment:
Only one line that I couldn't find a workaround for - src/Session/Store.js:93 valid-typeof rule
http://eslint.org/docs/rules/valid-typeof